tests/e2e/e2e_cli_util.go

```diff
@@ -167,3 +168,24 @@ func queryDelegation(endpoint string, validatorAddr string, delegatorAddr string
 
 	return delegationRes, nil
 }
+
+func queryDelegationWithdrawalAddress(endpoint string, delegatorAddr string) (disttypes.QueryDelegatorWithdrawAddressResponse, error) {
```

Comment:
queryDelegatorWithdrawalAddress?

Reply:
My preference is to have a more generic method that we pass the endpoint and any additional parameters. 
Otherwise we're repeating a lot.

Reply:
Sure, we still need a specific method that unmarshals the response to the corresponding response object but we could certainly abstract the rest of the code out to a helper.